Assets/MixedRealityToolkit/Definitions/Utilities/AxisFlags.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    /// <summary>
+    /// Flags used to represent a set of 3D axes
+    /// </summary>
+    [System.Flags]
+    public enum AxisFlags
```

Comment:
We have a few other enums for axis definitions as well. I wonder if we can reuse some or replace some existing ones with this.

This one is probably the best candidate for replacement, since it explicitly defines all combinations of 1 or 2 axes, which flags are good for!
https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.Utilities.PivotAxis.html

This one might be replaceable with this enum as well, though we'd have to account for the possibility of multiple values being set:
https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.BoundsExtensions.Axis.html

Reply:
Yeah, I can investigate similar enums and make sure we're reusing as much as possible.

Reply:
So I was going to have two separate enums for axes, one for flags (`AxisFlags`) and another as just a normal enum (`Axis` or something similar). One problem with changing the type of all of these enums is that in a lot of cases these are publicly modifiable properties, so it would be breaking for anyone who is setting these values in code.

I agree that deduplicating these enums (I found a few more than those you mentioned) is definitely something we want to do, but should we do so now if it may be breaking?